buildspec-linux-make-gcc-cov.yml

```
@@ -32,7 +32,7 @@ phases:
       - make -C jbmc/unit test
       - make -C jbmc/regression test
       - lcov --capture --directory . --output-file ./lcov.info
-      - VCS_PULL_REQUEST=$(echo $CODEBUILD_SOURCE_VERSION | sed 's/pr\///g')
+      - VCS_PULL_REQUEST=$(echo $CODEBUILD_SOURCE_VERSION | sed '/^pr\//!d;s/^pr\///')
```

A comment explaining the regex would be good

---

You can put a comment with a `#` on the line above I think
```suggestion
      # explanation of what the sed command does goes here 
      - VCS_PULL_REQUEST=$(echo $CODEBUILD_SOURCE_VERSION | sed '/^pr\//!d;s/^pr\///')
```

---

@thk123. I hope I addressed what you wanted. Let me know if you think I should improve this comment :-)